dev-libs/hiredict/hiredict-1.3.1.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Minimalistic C client library for the Redict database"
+HOMEPAGE="https://codeberg.org/redict/hiredict"
+SRC_URI="https://codeberg.org/redict/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+S=${WORKDIR}/${PN}
+
+LICENSE="BSD LGPL-3"
+SLOT="0/$(ver_cut 1-2)"
+KEYWORDS="~amd64"
+IUSE="shim examples ssl static-libs test"
+RESTRICT="!test? ( test )"
+
+DEPEND="ssl? ( dev-libs/openssl:= )"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	test? (
+		dev-db/redict
+		dev-libs/libevent
+	)
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.3.1-disable-network-tests.patch
+)
+
+src_prepare() {
+	default
+
+	# use GNU ld syntax on Solaris
+	sed -i -e '/DYLIB_MAKE_CMD=.* -G/d' Makefile || die
```

Comment:
This `sed` does nothing. The pattern is no longer present in the `Makefile`. It was changed in commit https://codeberg.org/redict/hiredict/commit/2cb203c1e99441172052f63afedd501ee2cedb65

Reply:
removed the sed